docker-compose.yaml

```diff
@@ -100,7 +100,7 @@ services:
       resources:
         limits:
           cpus: '2'
-          memory: '5g'
+          memory: '10g'
```

Comment:
When running 5 experiments together (even when inf was hosted outside the ray cluster) took more than 5g ram.